### PR TITLE
adding note about our yearly styleguide review

### PR DESCRIPTION
**Pull request by debugish**
adding note about our yearly styleguide review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,4 +51,4 @@ Set up your editor to automatically format code per our style guide:
 - **Sublime Text**
   - Install [JS Beautify](https://github.com/enginespot/js-beautify-sublime).
   - Update settings using [this config](./.jsbeautifyrc).
-  - Code is formatted automatically on save by default. You can turn this off in the .jsbeautifyrc file and format on demand using CNTL-OPTION-F (mac) CNTL-ALT-F (win).
\ No newline at end of file
+  - Code is formatted automatically on save by default. You can turn this off in the .jsbeautifyrc file and format on demand using CNTL-OPTION-F (mac) CNTL-ALT-F (win).
```
Comment (debugish) [inline README.md]: This is here because of: https://github.com/rei/code-style-guides/pull/10 ;p

Comment (robatron) [inline README.md]: Nit: I think "CNTL" should be "CTRL" b/c that's how it's labeled on most keyboards

File: README.md [hunk 2]
```diff
@@ -13,9 +13,12 @@
...
 Consistent code formatting increases readability, and reduces the overhead of understanding each other's code. A coder should be able to pick up any arbitrary file, and understand it quickly. Ideally, our codebase should look as though it were written by a single coder. You are not coding for yourself; you are coding for REI.
 
-### How do I propose a change?
+### What is the process for proposing a change?
 
-A developer who would like to propose a change to a code style guide must:
+The front-end development team holds an annual review event as a regular checkup to ensure the styles continue to fulfill their purpose, supplemented with ad hoc updates as new technologies, approaches, or styles gain traction with the team throughout the year.
+
+A developer who would like to propose a change to a code style guide can either
+wait for the annual styleguide review or purpose an an ad-hoc change:
```
Comment (robatron) [inline README.md]: Nit: Should probably remove this line break to make this sentence wrap consistently with the rest of the document

**Comment by debugish** [comment]
@petersgrandadventure & @mhewson Looking for your approvals to merge

**Comment by peripateticus** [comment]
Do we want to close or merge this?

**Pull request closed without merging**